Tests/SonarScanner.MSBuild.TFS.Test/BuildVNextCoverageReportProcessorTests.cs

```diff
@@ -58,9 +58,8 @@ public BuildVNextCoverageReportProcessorTests(TestContext testContext)
         alternateCoverageDir = Path.Combine(testResultsDir, "alternate", "In");
         directoryWrapper.Exists(alternateCoverageDir).Returns(true);
         propertiesFilePath = testDir + Path.DirectorySeparatorChar + "sonar-project.properties";
-        settings.BuildDirectory = testDir;
+        buildSettings.BuildDirectory = testDir;
         sut = new BuildVNextCoverageReportProcessor(converter, testLogger, fileWrapper, directoryWrapper);
```

Comment:
This has the same state problem as the other class. And `sut` is not even `readonly` here (another bad sign). Likely to be fixed, but not in this PR